ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA4MDg4OTk4NS4xNzQ1OTY2ODYxIn0sImFub25JZCI6ImM3YWQwNWQwNjcwZTg2YTUifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouYXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc2Fsb29uL3MtY2xhc3Mvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNzEiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI0NDIzMDAwODcwOTM4OTQxIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56525069acea <unknown>
#1 0x56525014b5f0 <unknown>
#2 0x565250131e86 <unknown>
#3 0x56525013079a <unknown>
#4 0x565250130eff <unknown>
#5 0x565250130e54 <unknown>
#6 0x56525014f7ff <unknown>
#7 0x5652501f49aa <unknown>
#8 0x5652501c2662 <unknown>
#9 0x5652501e8660 <unknown>
#10 0x5652501c2433 <unknown>
#11 0x56525018eea3 <unknown>
#12 0x56525018fb01 <unknown>
#13 0x56525065fb3b <unknown>
#14 0x565250663a21 <unknown>
#15 0x565250646c32 <unknown>
#16 0x565250664594 <unknown>
#17 0x56525062aeef <unknown>
#18 0x565250688d98 <unknown>
#19 0x565250688f76 <unknown>
#20 0x565250699b36 <unknown>
#21 0x7f574929caa4 <unknown>
#22 0x7f5749329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwODA4ODk5ODUuMTc0NTk2Njg2MSJ9LCJhbm9uSWQiOiJjN2FkMDVkMDY3MGU4NmE1In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjcxIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sIl90b29sc0V2ZW50TGlua0lkIjoiOTQzNjI3OTIwNTkwNTI2OSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56525069acea <unknown>
#1 0x56525014b5f0 <unknown>
#2 0x565250131e86 <unknown>
#3 0x56525013079a <unknown>
#4 0x565250130eff <unknown>
#5 0x565250130e54 <unknown>
#6 0x56525014f7ff <unknown>
#7 0x5652501f49aa <unknown>
#8 0x5652501c2662 <unknown>
#9 0x5652501e8660 <unknown>
#10 0x5652501c2433 <unknown>
#11 0x56525018eea3 <unknown>
#12 0x56525018fb01 <unknown>
#13 0x56525065fb3b <unknown>
#14 0x565250663a21 <unknown>
#15 0x565250646c32 <unknown>
#16 0x565250664594 <unknown>
#17 0x56525062aeef <unknown>
#18 0x565250688d98 <unknown>
#19 0x565250688f76 <unknown>
#20 0x565250699b36 <unknown>
#21 0x7f574929caa4 <unknown>
#22 0x7f5749329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMjMwIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJkZV9BVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjcxIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMDgwODg5OTg1LjE3NDU5NjY4NjEifSwiYW5vbklkIjoiYzdhZDA1ZDA2NzBlODZhNSJ9LCJwZXJmb3JtYW5jZSI6eyJzZGtQYXJzZVRpbWUiOjF9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7IlByb2R1Y3QiOnsiX2lkIjoiMjIzMCIsImRpbWVuc2lvbnMiOnsiQmF1bXVzdGVyU2hvcnQiOlsiMjIzMCJdfX19LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjIyNjIwNDMxNDM2NTk5MDMifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56525069acea <unknown>
#1 0x56525014b5f0 <unknown>
#2 0x565250131e86 <unknown>
#3 0x56525013079a <unknown>
#4 0x565250130eff <unknown>
#5 0x565250130e54 <unknown>
#6 0x56525014f7ff <unknown>
#7 0x5652501f49aa <unknown>
#8 0x5652501c2662 <unknown>
#9 0x5652501e8660 <unknown>
#10 0x5652501c2433 <unknown>
#11 0x56525018eea3 <unknown>
#12 0x56525018fb01 <unknown>
#13 0x56525065fb3b <unknown>
#14 0x565250663a21 <unknown>
#15 0x565250646c32 <unknown>
#16 0x565250664594 <unknown>
#17 0x56525062aeef <unknown>
#18 0x565250688d98 <unknown>
#19 0x565250688f76 <unknown>
#20 0x565250699b36 <unknown>
#21 0x7f574929caa4 <unknown>
#22 0x7f5749329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9zLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6IncyMjMiLCJ2ZWhpY2xlQk0iOiIyMjMwIiwidmVoaWNsZVBJSGlzdG9yeSI6IjIyMzAiLCJ0aW1lc3RhbXAiOjE3NDU5NjY4NjQ5NjIsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMDgwODg5OTg1LjE3NDU5NjY4NjEifSwiYW5vbklkIjoiYzdhZDA1ZDA2NzBlODZhNSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5hdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zYWxvb24vcy1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI3MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiOTM5MzI2MzExMjM4Njk2MyIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56525069acea <unknown>
#1 0x56525014b5f0 <unknown>
#2 0x565250131e86 <unknown>
#3 0x56525013079a <unknown>
#4 0x565250130eff <unknown>
#5 0x565250130e54 <unknown>
#6 0x56525014f7ff <unknown>
#7 0x5652501f49aa <unknown>
#8 0x5652501c2662 <unknown>
#9 0x5652501e8660 <unknown>
#10 0x5652501c2433 <unknown>
#11 0x56525018eea3 <unknown>
#12 0x56525018fb01 <unknown>
#13 0x56525065fb3b <unknown>
#14 0x565250663a21 <unknown>
#15 0x565250646c32 <unknown>
#16 0x565250664594 <unknown>
#17 0x56525062aeef <unknown>
#18 0x565250688d98 <unknown>
#19 0x565250688f76 <unknown>
#20 0x565250699b36 <unknown>
#21 0x7f574929caa4 <unknown>
#22 0x7f5749329c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/BFV1_attempt_2.png'